docs/topics/api.rst

```diff
@@ -182,7 +182,7 @@ does not require user authentication.
 
 
 .. _`retrieve a token`: https://developers.braintreepayments.com/javascript+python/reference/request/client-token/generate
-.. _`token generator`: https://solitude.readthedocs.org/en/latest/topics/braintree.html#generate-a-token
+.. _`token generator`: https://solitude.readthedocs.org/en/latest/topics/braintree.html#post--braintree-token-generate-
```

Comment:
yeah, they are definitely weird. I guess sphinx auto-generates anchors like this.